pages/integrations/github.md.erb

```diff
@@ -0,0 +1,58 @@
+# GitHub
+
+Buildkite can connect to your GitHub repository and use the Commit Status API to update the status of commits in Pull Requests.
+
+To complete this integration, you will need admin privileges for your GitHub repository.
+
+<%= toc %>
+
+## Connecting Buildkite and GitHub
+
+To connect your GitHub account open Buildkite’s Personal Settings, choose [Connected Apps](https://buildkite.com/user/connected-apps) then click the GitHub Connect button:
+
+<%= image "personal-settings.png", width: 1467/2, height: 565/2, alt: "Screenshot of the Buildkite Connected Apps screen" %>
+
+You'll be prompted to give permission for Buildkite to post status
+updates, and then redirected back to your Connected Apps page.
```

Comment:
How about adding in a screenshot of the auth screen here? Like we do in the [GitHub Enterprise docs](https://buildkite.com/docs/integrations/github-enterprise#step-3-connect-your-github-enterprise-account-to-buildkite)?

Reply:
I'd be inclined to leave it out, because there aren't really any options on that need explaining.

Reply:
Cool, sounds good!